app/components/search/SearchResults/SearchResults.tsx

```diff
@@ -12,35 +11,24 @@ import { useInstantSearch } from "react-instantsearch";
 import algoliasearchHelper from "algoliasearch-helper";
 import styles from "./SearchResults.module.scss";
 import ClearSearchButton from "../Refinements/ClearSearchButton";
+import { Loader } from "components/ui";
 
 const SearchResults = ({
   mobileMapIsCollapsed,
-  setAroundLatLng,
-  searchQuery,
 }: {
   mobileMapIsCollapsed: boolean;
-  setAroundLatLng: (latLng: { lat: number; lng: number }) => void;
-  searchQuery?: string | null;
 }) => {
   const {
     results: searchResults,
-  }: { results: algoliasearchHelper.SearchResults<SearchHit> } =
-    useInstantSearch();
-  const [centerCoords] = useState(null);
-  const [googleMapObject, setMapObject] = useState<google.maps.Map | null>(
-    null
-  );
-
-  useEffect(() => {
-    if (centerCoords && googleMapObject) {
-      googleMapObject.setCenter(centerCoords);
-    }
-    document.body.classList.add("searchResultsPage");
-
-    return () => {
-      document.body.classList.remove("searchResultsPage");
-    };
-  }, [googleMapObject, centerCoords]);
```

Comment:
The class setting is related to printing -- which we should better understand as a requirement later. Also I wasn't sure why setting the map center at this level mattered vs doing it in the `SearchMap` component on load. Doesn't seem to matter.